pkg/controllers/dynakube/istio/reconciler.go

```diff
@@ -2,14 +2,18 @@ package istio
 
 import (
 	"context"
+	"errors"
 	"net"
 
 	dynatracev1beta1 "github.com/Dynatrace/dynatrace-operator/pkg/api/v1beta1/dynakube"
 	dtclient "github.com/Dynatrace/dynatrace-operator/pkg/clients/dynatrace"
 	"github.com/Dynatrace/dynatrace-operator/pkg/controllers/dynakube/connectioninfo/activegate"
 	"github.com/Dynatrace/dynatrace-operator/pkg/controllers/dynakube/connectioninfo/oneagent"
+	"github.com/Dynatrace/dynatrace-operator/pkg/util/conditions"
 	"github.com/Dynatrace/dynatrace-operator/pkg/util/kubeobjects/labels"
-	"github.com/pkg/errors"
+	"github.com/Dynatrace/dynatrace-operator/pkg/util/timeprovider"
+	errorshelper "github.com/pkg/errors"
```

Comment:
just to be consistent:

we tend to alias the `errors` to `goerrors`
and leave `github.com/pkg/errors` as `errors`

https://github.com/Dynatrace/dynatrace-operator/blob/7d61cb9fed001f98c64157b193f4bc25eca742cf/pkg/controllers/dynakube/controller.go#L5

Reply:
fixed in cad3eb41